doc/classes/GeometryInstance.xml

```diff
@@ -68,6 +68,10 @@
 			The GeometryInstance's min LOD margin.
 			[b]Note:[/b] This property currently has no effect.
 		</member>
+		<member name="material_overlay" type="Material" setter="set_material_overlay" getter="get_material_overlay">
+            The material overlay for the whole geometry.
+            If a material is assigned to this property, it will be rendered on top of any other active material for all the surfaces.
```

Comment:
Use tab indentation as expected by the editor:

```suggestion
			The material overlay for the whole geometry.
			If a material is assigned to this property, it will be rendered on top of any other active material for all the surfaces.
```